lib/hsa/mcwamp_hsa.cpp

```diff
@@ -1478,7 +1470,7 @@ class HSAQueue final : public KalmarQueue
             if (asyncOp != nullptr) {
                 hsa_signal_t signal = *(static_cast <hsa_signal_t*> (asyncOp->getNativeHandle()));
                 if (signal.handle) {
-                    hsa_signal_value_t v = hsa_signal_load_relaxed(signal);
+                    hsa_signal_value_t v = hsa_signal_load_acquire(signal);
```

Comment:
Unless I am mistaken, this is deprecated, with `hsa_signal_load_scacquire` being the current / preferred form.